### ISSUE TITLE
Add Logging

### ISSUE THREAD

**Issue by toastdriven** [toastdriven/itty3#5]
## Problem

`logging` is a pretty great idea. Finish the `logging` branch, which currently seems to be blocked on configuration.

## Setup

Python version: 3.7.4
itty3 version (`itty3.get_version(full=True)`): 1.0.0-alpha-2
OS: All
Any other libraries/setup: N/A

**Comment by toastdriven** [comment]
Merged into develop in SHA: 8e72650
